### DISCUSSION TITLE
Referrer policy:

### DISCUSSION THREAD

**Discussion by craigfrancis** [alphagov/govuk-design-system#2931] [Exit this page]
I would expect this link to not inform the destination website where the user came from.

i.e. the link should set the `rel=noreferrer` or `referrerpolicy="no-referrer"` attributes:

    <a href="https://www.bbc.co.uk/weather" rel="noreferrer">Exit this page</a>
  
    <a href="https://www.bbc.co.uk/weather" referrerpolicy="no-referrer">Exit this page</a>

See the [Referrer Policy Specification](https://www.w3.org/TR/referrer-policy/#referrer-policy-delivery).

Support for `rel="noreferrer"` is from Chrome 16, Firefox 33, Safari 5.

Support for `referrerpolicy="?"` is from Chrome 51, Firefox 50, and Safari 14. While this version gives more control (e.g. `referrerpolicy="strict-origin"`), I doubt this is needed.

<img src="https://github.com/alphagov/govuk-design-system/assets/1296695/4aba50ac-071d-4082-9bbb-330a63878c9b" alt="Support Table from MDN, link below" />

[Compatibility Table](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#browser_compatibility)

**Comment by querkmachine** [comment]
Hi @craigfrancis. We've already got an open issue for adding this to Exit this Page in the next iteration. Feel free to add any extra comments over there. https://github.com/alphagov/govuk-frontend/issues/3921 

Thanks for raising it!
